components/Sections/MainHeader.tsx

```
@@ -5,33 +5,107 @@ import { Button } from '@/components/ui/button';
+  subtitle: `font-sans text-lg md:text-xl mb-8 max-w-2xl 
+             text-primary-foreground`,
+  
+  ctaButton: `bg-gradient-to-r from-primary to-primary/80 
+              hover:from-primary/80 hover:to-primary 
+              text-primary-foreground shadow-lg`
+} as const;
+
+interface MainHeaderProps {
+  // Future extensibility - props can be added here
+}
+
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)